backend/src/main/java/de/neuefische/backend/model/PumpTimer.java

```diff
@@ -9,4 +9,6 @@
 
     private float minutes;
```

Comment:
not very expressive variable naming. minutes of what?

Reply:
It's called pumpRunTimeInMinutes now